lib/tasks/gettext.rake

```diff
@@ -52,6 +52,68 @@ namespace :gettext do
            files_to_translate.join(' '))
   end
 
+  def metadata_pot_file_path
+    File.join(locale_path, GettextSetup.config['project_name'] + '_metadata.pot')
+  end
+
+  def generate_new_pot_metadata
+    open(metadata_pot_file_path, 'w') do |f|
+      f << "#\n"
+      f << "#, fuzzy\n"
+      f << "# msgid \"\"\n"
+      f << "# msgstr \"\"\n"
+      f << "\"Project-Id-Version: PACKAGE VERSION\\n\"\n"
+      f << "\"Report-Msgid-Bugs-To: \\n\"\n"
+      f << "\"POT-Creation-Date: #{current_time} \\n\"\n"
+      f << "\"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\\n\"\n"
+      f << "\"Last-Translator: FULL NAME <EMAIL@ADDRESS>\\n\"\n"
+      f << "\"Language-Team: LANGUAGE <LL@li.org>\\n\"\n"
+      f << "\"MIME-Version: 1.0\\n\"\n"
+      f << "\"Content-Type: text/plain; charset=UTF-8\\n\"\n"
+      f << "\"Content-Transfer-Encoding: 8bit\\n\"\n"
+      f << "\"X-Generator: Translate Toolkit 2.0.0\\n\"\n"
+      f << "#\n"
+
+      metadata = load_metadata_information
+
+      if metadata.key?('summary')
+        f << "#. metadata.json\n"
+        f << "#: .summary\n"
+        f << "msgid \"#{metadata_summary}\"\n"
+        f << "msgstr \"\"\n"
+        f << "\n"
+      end
+      if metadata.key?('description')
+        f << "#. metadata.json\n"
+        f << "#: .description\n"
+        f << "msgid \"#{metadata_description}\"\n"
+        f << "msgstr \"\"\n"
+        f << "\n"
+      end
+    end
+  end
+
+  def load_metadata_information
+    metadata_file = 'metadata.json'
```

Comment:
To avoid this being called so many times, opening the file each time, I suggest doing something like this:

```
@metadata ||= begin
  metadata_file = 'metadata.json'
  file = open(metadata_file)
  json = file.read
  JSON.parse(json)
end
```